components/chef-ui-library/package-lock.json

```diff
@@ -5290,7 +5290,7 @@
         },
         "readable-stream": {
           "version": "1.0.34",
```

Comment:
These downgrades from https to http are unexpected. Suggest revert these changes here and start a fresh issue. Some relevant links (the second one has a suggested way to clean it up):
https://github.com/npm/npm/issues/20719
https://npm.community/t/npm-install-changes-package-lock-json-resolve-from-http-to-https/6899

Reply:
thanks for those fixes - appears to have worked.  Are you suggesting I open an issue with npm?

Reply:
No, I was suggesting to fix it on our side separately from this PR... but if the linked fix already cured it for you, nothing more to do. 😁